sympy/solvers/tests/test_inequalities.py

```diff
@@ -323,16 +340,24 @@ def test_issue_10198():
         -1 + 1/abs(1/x - 1) < 0) == Or(
         And(-oo < x, x < 0), And(S(0) < x, x < S(1)/2)
         )
-    raises(NotImplementedError, lambda: reduce_inequalities(abs(1/sqrt(x)) - 1, x))
+    assert reduce_inequalities(abs(1/sqrt(x)) - 1, x) == Or(
+        And(-oo < x, x < 0), Eq(x, 1))
```

Comment:
> Ans is +1, -1, right ?

No. `-1` can't be an answer.

> `abs(1/sqrt(-1))` is one.
Since, `1/sqrt(-1)` is a complex number( iota `I` ).
Taking its `Abs` returns its magnitude i.e. 1.

We solve inequalities in the real domain only.